src/go/go-binding/capi.go

```diff
@@ -434,6 +436,20 @@ func main() {
 	server_http_port := 11338
 	// skip_server_create := 0;
 
+	var processName string
+	processName = "CPU"
+	if len(os.Args) == 2 {
+		processName = os.Args[1]
+		if !(processName == "CPU" || processName == "GPU.1") {
+			fmt.Printf("invalid input processName: %v\n", processName)
+			fmt.Println("Please enter CPU or GPU.1")
+			return
+		}
+	}
+
+	target_device := "\"target_device\": \"" + processName + "\"},"
+	util.ReplaceConfigLine("\"target_device\":", target_device, modelConfigPath)
```

Comment:
please change to use flags, as this positional argument maynot be working well in the long run